deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -97,7 +97,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: hostpath
-          image: quay.io/k8scsi/hostpathplugin:v1.2.0
+          image: k8s.gcr.io/sig-storage/hostpathplugin:v1.2.0
```

Comment:
This image does not exist.  Let's wait for the new csi hostpath driver image to be cut.

Reply:
all images will go to k8s.gcr.io in the future?

Reply:
You can use hostpath driver 1.4.0